app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
+
+    def wrapper(*args) -> None:
```

Comment:
The `wrapper` function is currently defined to return `None`, but it should return the result of the function call. Change the return type from `None` to the appropriate type, or remove the type hint if it's not necessary.